src/arrayMethodSort.js

```diff
@@ -3,9 +3,33 @@
 /**
  * Implement method Sort
  */
+const compareFunction = function(a, b) {
+  const stringA = a.toString();
+  const stringB = b.toString();
+
+  return stringA > stringB ? 1 : stringA === stringB ? 0 : -1;
```

Comment:
Якщо не помиляюсь, функціонально нема ніякої різниці, повертаєш ти 0 чи -1. Так що можна просто 
```suggestion
  return stringA > stringB ? 1  :  -1;
```